internal/collectors/lscpu.go

```diff
@@ -0,0 +1,36 @@
+package collectors
+
+import (
+	"bufio"
+	"bytes"
+	"strings"
+
+	"github.com/SUSE/connect-ng/internal/util"
+)
+
+func lscpu() (map[string]string, error) {
+	output, err := util.Execute([]string{"lscpu"}, nil)
```

Comment:
We should consider to [memoize](https://fullstackdojo.medium.com/memoization-in-golang-a946acd10829) the call to `lscpu` here next when it gets called for multiple collectors like cpu+hypervisor.

Reply:
I add it as nice to have and we should revisit this when we looking into the hypervisor detection.